pkg/controller/volume/scheduling/scheduler_binder.go

```diff
@@ -559,20 +559,20 @@ func (b *volumeBinder) checkBindings(pod *v1.Pod, bindings []*BindingInfo, claim
 	_, err = b.podLister.Pods(pod.Namespace).Get(pod.Name)
 	if err != nil {
 		if apierrors.IsNotFound(err) {
-			return false, fmt.Errorf("pod %q does not exist any more", podName)
+			return false, fmt.Errorf("pod does not exist any more: %w", err)
```

Comment:
keep podName as a part of the message? or does having that as a part of the string cause unwrapping issues?

Reply:
the error from the API already includes the pod name. Other than that, we should add the metadata to the error when we are logging it. See #94697